src/components/ArticleCard.jsx

```diff
@@ -1,13 +1,23 @@
 import "./styles.css";
+import { Link } from "react-router-dom";
 
-const ArticleCard = ({ title, articleBody, commentCount, votes }) => {
+
+const ArticleCard = ({
+  article_id,
+  title,
+  articleBody,
+  commentCount,
+  votes
+}) => {
   return (
     <div className="articleCard">
-      <h3>{title}</h3>
-      <p>
-        VOTES: {votes} COMMENTS: {commentCount}
-      </p>
-      <p className="articlePreview">{articleBody}</p>
+      <Link to={`/articles/${article_id}`}>
+        <h3>{title}</h3>
+        <p>
+          VOTES: {votes} COMMENTS: {commentCount}
```

Comment:
When I see the vote counts on the article cards, they're always zero - is this correct?